test/commands/clear-account.test.js

```diff
@@ -0,0 +1,43 @@
+const { expect, test } = require("@oclif/test");
+const sinon = require("sinon");
+const lambda = require("../../src/lib/lambda");
+const iam = require("../../src/lib/iam");
+const apigw = require("../../src/lib/apigw");
+const s3 = require("../../src/lib/s3");
+const cf = require("../../src/lib/cloudformation");
+const versionCheck = require("../../src/lib/version-check");
+
+jest.mock("../../src/lib/version-check");
+jest.mock("../../src/lib/cloudformation");
+jest.mock("../../src/lib/s3");
+jest.mock("../../src/lib/apigw");
+jest.mock("../../src/lib/iam");
+jest.mock("../../src/lib/lambda");
+describe("User forces clear account", () => {
+	beforeEach(() => {
+		versionCheck.checkVersion.mockResolvedValue(null);
+		s3.getBucketCount.mockResolvedValue(1);
+		s3.deleteAllBuckets.mockResolvedValue([{ status: "fail" }]);
+		apigw.getAllApiGwCount.mockResolvedValue(1);
+		apigw.deleteAllApiGw.mockResolvedValue([{ status: "success" }]);
+		iam.getAllRolesCount.mockResolvedValue(1);
+		iam.deleteAllRoles.mockResolvedValue([{ status: "success" }]);
+		cf.deleteAllStacks.mockResolvedValue([{ status: "success" }]);
+		cf.getAllStacksCount.mockResolvedValue(1);
+		lambda.getAllFunctionsCount.mockResolvedValue(0);
+		lambda.deleteAllFunctions.mockResolvedValue([{ status: "success" }]);
+	});
+	afterEach(() => {
+		sinon.restore();
```

Comment:
you don't need this anymore, right? since you're mocking with `jest`, it has a `restoreAllMocks` method that does the same thing